templates/common/on-prem/files/keepalived.yaml

```diff
@@ -70,6 +70,13 @@ contents:
         - -c
         - |
           #/bin/bash
+          sigterm_handler()
+          {
+            if pid=$(pgrep -o keepalived); then
```

Comment:
`killall /usr/sbin/keepalived` could simplify this a bit (has the same effect of `grep -o...`